template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/CmTemplateValidatorTest.java

```diff
@@ -3,12 +3,14 @@
 import static com.sequenceiq.cloudbreak.TestUtil.hostGroup;
 import static java.util.stream.Collectors.toSet;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.when;
 
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Set;
 
 import org.junit.Test;
```

Comment:
Can you please use `org.junit.jupiter.api.Test` instead?

Reply:
Also can you please use `@ExtendWith(MockitoExtension.class)` instead of `@RunWith(MockitoJUnitRunner.class)`.

Reply:
It is an existing test class. I would not like to change it, instead, we should create a ticket to remove all junit4 tests and rewrite them to junit5.